frontend/src/styles/components/Maps.scss

```diff
@@ -6,3 +6,16 @@ This file contains the style overrides for ArcGIS Map components
 path[path="M25,0 C60,0, 60,50, 25,50 C-10,50, -10,0, 25,0"] {
   @include pulsateAnimation;
 }
+
+.minePopUpText {
+  line-height: 1.5;
+  font-size: 1rem;
+}
+
+.leftMapPopUpCol {
+  width: 10%;
+}
+
+.mineMapPopUpButton {
+  float: right;
+}
```

Comment:
highly recommend using flex over the float property.

if you want something to be positioned right, wrap the button in a parent div
```
<div className="mineMapPopUpButton">
<button>click me</button>
</div>
```
and add update CSS to be: 

```
mineMapPopUpButton {
 @ include: flex;
justify-content: flex-end;
}
```

That should do the trick!

Reply:
done